Library/Formula/kettle.rb

```diff
@@ -0,0 +1,61 @@
+class Kettle < Formula
+  homepage "http://community.pentaho.com/projects/data-integration/"
+  url "https://downloads.sourceforge.net/project/pentaho/Data%20Integration/5.0.1-stable/pdi-ce-5.0.1-stable.zip"
+  sha1 "c34fa3dbe8b75280fd3f7ddcaf609acbcdd2ed78"
+
+  def install
+    rm_rf Dir["*.{bat}"]
+    libexec.install Dir["*"]
+
+    (etc+"kettle").install libexec+"pwd/carte-config-master-8080.xml" => "carte-config.xml"
+    (etc+"kettle/.kettle").install libexec+"pwd/kettle.pwd"
+    (etc+"kettle/simple-jndi").mkpath
+
+    (var+"log/kettle").mkpath
+
+    # We don't assume that carte, kitchen or pan are in anyway unique command names so we'll prepend "pdi"
+    %w[carte kitchen pan].each do |command|
+      (bin+"pdi#{command}").write_env_script libexec+"#{command}.sh", :BASEDIR => libexec
+    end
+  end
+
+  def plist; <<-EOS.undent
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN"
+    "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+      <dict>
+        <key>Label</key>
+        <string>#{plist_name}</string>
+        <key>ProgramArguments</key>
+        <array>
+          <string>#{bin}/pdicarte</string>
+          <string>#{etc}/kettle/carte-config.xml</string>
+        </array>
+        <key>EnvironmentVariables</key>
+        <dict>
+          <key>KETTLE_HOME</key>
+          <string>#{etc}/kettle</string>
+        </dict>
+        <key>StandardOutPath</key>
+        <string>#{var}/log/kettle/carte.log</string>
+        <key>StandardErrorPath</key>
+        <string>#{var}/log/kettle/carte.log</string>
+        <key>RunAtLoad</key>
+        <true/>
+      </dict>
+    </plist>
+    EOS
+  end
+
+  test do
+    system "launchctl", "load", "#{prefix}/#{plist_name}.plist"
+    sleep 10
+    system "curl", "--fail", "http://cluster:cluster@localhost:8080"
+
+    chmod 0755, "#{libexec}/samples/jobs/shell for every row/shelltest.cmd"
+    system "pdikitchen", "-file=#{libexec}/samples/jobs/shell\\ for\\ every\\ row/shell\\ for\\ every\\ row.kjb", "-level=Minimal"
+
+    system "pdipan", "-file=/usr/local/opt/kettle/libexec/samples/transformations/Encrypt\\ Password.ktr", "-level=Minimal"
```

Comment:
If either of the `pdi` commands can be run without the cluster up then just run one of them alone as a simpler test. Also, make sure you do `system "#{bin}/pdipan` and use `HOMEBREW_PREFIX` instead of `/usr/local`. Thanks!